### PR TITLE
Display rating list

### PR DESCRIPTION
**Pull request by jeinygroove**

### CODE REVIEW

File: src/main/webapp/src/components/ratingList.js [hunk 1]
```diff
@@ -31,71 +33,74 @@ function EnumeratedList(props) {
...
+  }
+
   render() {
     const classes = this.props.classes;
+    const topSongs = this.state.topSongs;
+
+    if (this.state.error) {
+      return (
+        <div>
+          <p>{this.state.error.message}</p>
```
Comment (RazvanBudacaTechquarter) [inline src/main/webapp/src/components/ratingList.js]: Can you give an example of error? I'm thinking that maybe the user will have no idea what it actually says

**Review by RazvanBudacaTechquarter — CHANGES_REQUESTED** [review]
All good besides the shady error message.

File: src/main/webapp/src/components/ratingList.js (hunk 1 above)
Comment (jeinygroove) [reply src/main/webapp/src/components/ratingList.js]: It depends on the servlet actually. Usually, it displays error that the servlet can send.

Comment (RazvanBudacaTechquarter) [reply src/main/webapp/src/components/ratingList.js]: Maybe we could print this error to console so we know what's happening, but show to the user only something like
`Oops, something went wrong, please try again`

**Review by RazvanBudacaTechquarter — APPROVED** [review]

**Review by sashapff — APPROVED** [review]

File: src/main/webapp/src/components/ratingList.js [hunk 2]
```diff
@@ -31,71 +33,75 @@ function EnumeratedList(props) {
...
+        })
+      );
+  }
+
   render() {
     const classes = this.props.classes;
+    const topSongs = this.state.topSongs;
+
+    if (this.state.error) {
+      console.log(this.state.error.message);
```
Comment (jgaillard85) [inline src/main/webapp/src/components/ratingList.js:80]: In the future, it might good to think of a proper way to log messages on the server side

**Review by jgaillard85 — APPROVED** [review]

**Pull request merged**